src/sass/item-tile/item-tile.scss

```diff
@@ -0,0 +1,79 @@
+.item-tile {
+    display: grid;
+    gap: var(--spacing-200);
+    grid-area: body;
+    grid-template-areas:
+        "header"
+        "body";
+    grid-template-rows: auto 1fr;
+}
+
+.item-tile--list-view {
+    display: flex;
+    flex-direction: row;
+    gap: var(--spacing-100);
+}
+
+.item-tile__header {
+    grid-area: header;
+}
+
+.item-tile--list-view .item-tile__header {
+    max-width: 200px;
+}
+
+.item-tile__body {
+    display: grid;
+    grid-area: body;
+    grid-template-areas:
+        "aligned-section-1"
+        "aligned-section-2"
+        "aligned-section-3";
```

Comment:
Not sure I really like the section 1, 2, 3. Seems arbitrary. I can see it being fine in case we need another section, but I don't see that necessary
Can we do maybe: section-primary, section-secondary, section-tertiary?
Or section-main, section-sub, section-last? 
Or section-title, section-description, section-additional-info?
Or a combination?
I think I prefer it to be much more descriptive of what each section is, rather than 1, 2, and 3

Reply:
Good call. Changing...